CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# v1.46.0
+
+:new: What's new:
+- Feature: Repository Search by Substring (#8417)
+- adjusted dockerfile so that build args can be provided (#8420)
```

Comment:
I think this is unrelated to the version you're releasing. i.e. lakeFS users won't enjoy it, lakeFS developers/contributors will.

Reply:
removed it